### ISSUE TITLE
Wrong version returned by pljson_parser.get_version

### ISSUE THREAD

**Issue by akinet** [pljson/pljson#123]
Hi.

I have downloaded pljson-2.1.0.zip and I have seen that pljson_parser.get_version is not returning 'PL/JSON v2.1.0':

```
  function get_version return varchar2 as
  begin
    return 'PL/JSON v2.0.0';
  end get_version;
```

**Comment by jsumners** [comment]
Thank you for reporting this. I'll need to determine a way to handle this while building a release.

Out of curiosity, do you depend on this method? I'm wondering if we even need to include a `get_version` method.

**Comment by akinet** [comment]
I think is good to have a way to know version of installed code. For example, I have found this wrong version because I have been looking in my server for the number of the installed version (1.0.,4). I wanted to know which "uninstall.sql" script should I use to remove the old version before to install the last one. I am updating because the server has been updated to Oracle 12.2. If the "uninstall.sql" script is different from version to version is worth using the proper one. 
Perhaps, a function with a hardcoded version is not the best option. It should be automatically tagged with the right version.
But if, at this moment, there is no other option it's a good option.
